noodles-editor/src/noodles/storage.ts

```diff
@@ -11,6 +12,14 @@ import {
 } from './utils/filesystem'
 import { EMPTY_PROJECT, type NoodlesProjectJSON, safeStringify } from './utils/serialization'
 
+// Pre-load all example asset URLs at build time using import.meta.glob
+// This creates a lookup map: '../examples/project-name/file.ext' -> URL string
+const exampleAssetUrls: Record<string, string> = import.meta.glob('../examples/**/*', {
+  eager: true,
```

Comment:
Should we put all of these globs in one global file? To reduce the number of places they're called